client/components/Background.jsx

```diff
@@ -4,24 +4,21 @@ import throttle from 'lodash/throttle';
 export default function Background() {
   const [mousePosition, setMousePosition] = useState({ left: 0, top: 0 });
   useEffect(() => {
-    addEventListener();
-    return () => removeEventListener();
-  });
-  const addEventListener = () => {
     document.addEventListener('mousemove', handleMouseMove);
-  };
-
-  const removeEventListener = () => {
-    document.addEventListener('mousemove', handleMouseMove);
-  };
-
-  const handleMouseMove = useCallback(throttle(event => { setMousePosition({ left: event.clientX, top: event.clientY }); }, 1), []);
+    return () => document.removeEventListener('mousemove', handleMouseMove);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I like to add a comment on my lint disable lines. You do it by putting ` -- ` followed by the comment:
```
// eslint-disable-next-line react-hooks/exhaustive-deps -- Run only once on mount
```